resources/views/livewire/project/shared/destination.blade.php

```diff
@@ -78,12 +78,10 @@ class="absolute bg-error -top-1 -left-1 badge "></div>
     </div>
     @if ($resource->getMorphClass() === 'App\Models\Application' && data_get($resource, 'build_pack') !== 'dockercompose')
         <div class="flex flex-col gap-2">
-            @if ($resource->persistentStorages()->count() > 0)
+            @if ($resource->fileStorages()->count() > 0)
```

Comment:
In my PR I'd called
```
->get()->filter(fn($storage) => $storage->content)
```
because iirc `fileStorages` targets both file mounts and directory mounts. We only want to check for file mounts here and for those prevent adding additional servers, because those provide an option in the UI to set the contents of the file. Directory mounts are working fine in multi-server setups.